nexus-writer/src/main.rs

```diff
@@ -115,13 +119,14 @@ struct Cli {
 async fn main() -> anyhow::Result<()> {
     let args = Cli::parse();
 
+    debug!("{args:?}");
```

Comment:
Should this be here?

Reply:
That's a relic from past logging, I'll remove it in a nexus-writer specific PR.